lib/active_admin/view_helpers/download_format_links_helper.rb

```diff
@@ -7,11 +7,15 @@ def build_download_format_links(formats = self.class.formats)
         div class: "download_links" do
           span I18n.t('active_admin.download')
           formats.each do |format|
-            a format.upcase, href: url_for(params: params, format: format)
+            a format.upcase, href: download_url_for(params: params, format: format)
           end
         end
       end
 
+      def download_url_for(options = {})
+        url_for(options)
+      end
+
```

Comment:
This method will be overridden by ActiveAdmin::Views::PaginatedCollection when included by the class, so you can omit this.  However, I thought it should be defined here just in case this helper module gets included by other view component classes.